test/stablehlo/test_stablehlo_save_load.py

```diff
@@ -109,6 +122,18 @@ def test_save_load2(self):
       result = program2(*inputs).detach().cpu()
     self.assertTrue(torch.allclose(model(*inputs), result))
 
+  def test_save_load2_without_saving_weights(self):
+    model = ElementwiseAdd()
+    inputs = model.get_random_inputs()
+    options = StableHLOExportOptions()
+    options.override_tracing_arguments = inputs
+    options.save_weights = False
+    with tempfile.TemporaryDirectory() as tempdir:
+      save_torch_model_as_stablehlo(model, inputs, tempdir, options)
+      program2 = StableHLOGraphModule.load(tempdir)
+      result = program2(*inputs).detach().cpu()
+    self.assertTrue(torch.allclose(model(*inputs), result))
```

Comment:
Change these 2 lines to:
1. check that tempdir dont have weights
2. check that program2._bundle.state_dict is empty (no weights)

Reply:
thanks, updated